modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -5,8 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+The *Topology View* in the Developer perspective of the web console provides the following options to interact with the application and the components:
 
+* Use the *Shrtcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
```

Comment:
*Shortcuts*
there is no need for the word 'listed'
upper right - see ISG pg206, chpt 7

Reply:
Fixed